cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -127,6 +134,34 @@ data QueryInMode mode result where
   QuerySystemStart
     :: QueryInMode mode SystemStart
 
+  QueryChainBlockNo
+    :: QueryInMode mode (WithOrigin BlockNo)
+
+  QueryChainPoint
+    :: QueryInMode mode (Point mode)
+
+data Point mode where
+  Point
+    :: ( ConsensusBlockForMode mode ~ blk
+       )
```

Comment:
Why not ` :: (ConsensusBlockForMode mode ~ blk)`? Seems weird to newline for a bracket.

Reply:
This code is deleted.